en/computer_vision/collision_prevention.md

```diff
@@ -12,25 +12,26 @@ It can be enabled for multicopter vehicles in [Position mode](../flight_modes/po
 
 *Collision Prevention* is enabled/configured on PX4 by setting the parameter for minimum allowed distance ([MPC_COL_PREV_D](../advanced_config/parameter_reference.md#MPC_COL_PREV_D)).
 
-The feature requires obstacle information from either an external system (sent using the MAVLink [OBSTACLE_DISTANCE](https://mavlink.io/en/messages/common.html#OBSTACLE_DISTANCE) message) or a [distance sensor](../sensor/rangefinders.md) connected to the flight controller.
+The feature requires obstacle information from an external system (sent using the MAVLink [OBSTACLE_DISTANCE](https://mavlink.io/en/messages/common.html#OBSTACLE_DISTANCE) message) and/or a [distance sensor](../sensor/rangefinders.md) connected to the flight controller.
 
 > **Note** Multiple sensors can be used to get information about, and prevent collisions with, objects *around* the vehicle. 
   If multiple sources supply data for the *same* orientation, the system uses the data that reports the smallest distance to an object.
 
-The vehicle starts braking as soon as it detects an obstacle.
-The velocity setpoint towards the obstacle is reduced linearly such that it is set to zero at the point when the vehicle reaches the minimum allowed distance.
+Collision prevention determines the desired speed using the minimum distance from *the set of all included* sensors in the commanded direction (where the "included" set of sensors is tuned using [MPC_COL_PREV_ANG](#angle_tuning)).
```

Comment:
This sounds not entirely correct to me. I would go for something like :

The data from all sensors is fused into a range scan, which is the input to the collision prevention algorithm. The parameter MPC_COL_PREV_ANG determines which part of the of the range scan is used for the calculation of the velocity limit. The considered data from the range scan is centered around the commanded direction and spreads by MPC_COL_PREV_ANG to both sides.

Reply:
Thanks for those clarifications. I was assuming a simplistic algorithm, but of course you use data from across the considered (what I have called "included") data.

Reply:
@baumanta THanks! The term "range scan" is a bit domain specific - not sure if there is something more mainstream or if we should define it?

I assume this means a 360 degree map around the vehicle containing all sensor input (ie with indications of distance to obstacle, out of range, or no sensor here)

What I put in here was ....

> The data from all sensors is fused into a range scan (a 360 degree map of the sensor data/state from around the vehicle), which is then input to the collision prevention algorithm.
The collision prevention algorithm uses/considers only part of the range scan data in the calculation of the velocity limit - the data within two [MPC_COL_PREV_ANG](../advanced_config/parameter_reference.md#MPC_COL_PREV_ANG) degree arcs centered around the commanded direction.
For more information see [MPC_COL_PREV_ANG Angle Tuning](#angle_tuning).